### ISSUE TITLE
Make background white again

### ISSUE THREAD

**Issue by marcoambrosini** [nextcloud/tasks#2403]
### Describe the solution you'd like

I think we should go back to a white background. Looks better and it's consistent with other nextcloud apps

## **Now:**
![Screenshot from 2023-11-30 05-49-23](https://github.com/nextcloud/tasks/assets/26852655/16531fc5-ca54-4a63-a373-cc64f5c64150)

## **Proposed change:**
![Screenshot from 2023-11-30 05-48-51](https://github.com/nextcloud/tasks/assets/26852655/318775c1-1a4d-44a3-a7de-818829c75a2a)

**Comment by raimund-schluessler** [comment]
I think it looks better with the light gray background. It also gives a bit more contrast to the tasks.

**Comment by marcoambrosini** [comment]
@raimund-schluessler I don't think that that contrast is needed.  The page is easily readable without a gray background, and I think it doesn't make much sense that we have this in only this one app. 
@nextcloud/designers ping again, what do you think?

**Comment by szaimen** [comment]
I also think that white background is cleaner. What about making this configurable if we cannot agree on it?

**Comment by marcoambrosini** [comment]
ping @jancborchardt @nimishavijay

**Comment by jancborchardt** [comment]
I'd agree that the white background is better:
- Looks more modern
- Fits with the rest of the apps
- The tasks are already well separated with their container lines
